docs/tutorials/configure/pet-photographer.md

```diff
@@ -0,0 +1,290 @@
+tags: ["vision", "filter", "camera", "detector", "services"]
+aliases:
+  - /tutorials/pet-photographer
+  - /tutorials/filter-modular-component
+authors: ["Sky Leilani"]
+languages: []
+viamresources: ["vision", "camera"]
+level: "Beginner"
+date: "2023-09-17"
+# updated: ""
+cost: "0"
+no_list: true
+weight: 3
+---
+
+Smart machines are an integral part of our daily lives.
+From our phones to traffic lights, these machines rely on their ability to process and respond to data.
+However, as the amount of data collected by these devices continues to grow, it becomes important to not only process it, but also to organize and manage it effectively.
+
+The filter {{< glossary_tooltip term_id="module" text="module" >}} allows you to filter image data from a robot based on whether a certain condition is met.
+For example, if a specified color is present.
+This is useful for smart machines like bird feeders, which you could program to take a picture every five seconds and then store only pictures containing the distinctive red color that robins have.
+Another use case is to make a pet photographer:
+
+1. Set up a webcam in a location where your pet is likely to appear in frame and use the data management service to periodically take pictures and sync them to the [Viam's cloud](/services/data/#cloud-sync).
+2. Attach a colored object, like a blue collar, to your pet.
+3. Set up the color filter module, which will sift images and only store them if your pet and their easily identifiable colored object is present.
+
+{{<imgproc src="/tutorials/pet-photographer/data-example.png" resize="700x" declaredimensions=true alt="Dog in blue collar in the camera's live feed">}}
+
+While the color filter module you use in this tutorial selects image data from a camera, these same principles can be applied to various components, including for filtering [sensor](https://github.com/viam-labs/modular-filter-examples/tree/main/sensorfilter) data.
+The filter modular component allows you to selectively [store data](/services/data/#data-capture) when that data meets certain conditions you set.
+This can help you to avoid sifting through unwanted data captures and ensures that only the data you're interested in gets stored in Viam's cloud.
+
+## Hardware Requirements
+
+For this tutorial you'll need the following hardware components:
+
+- A computer
+- A webcam or external camera
+- A colored object, like a blue collar for enhanced accuracy _(optional)_
+
+{{< alert title="Tip" color="tip" >}}
+In this tutorial, the camera is configured to identify and filter images with the color blue, as it is less common in many environments, including mine.
+If your pet already has a distinct color that is different from their environment, you can also configure your camera to use that color to identify pictures of your pet.
+{{< /alert >}}
+
+## Set up
+
+Here's how to get started:
+
+1. Install [Go](https://go.dev/dl/) on your robot's computer.
```

Comment:
Doesn't need to be here - but somewhere in this tutorial we should remind users to update their server, and warn that any version < 0.8.0 will not have the filtering functionality.